src/components/UI/Display/MAPI/clusters/ClusterDetail/ClusterDetailWidgetOptionalValue.tsx

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import styled from 'styled-components';
+import NotAvailable from 'UI/Display/NotAvailable';
+import RefreshableLabel from 'UI/Display/RefreshableLabel';
+
+import ClusterDetailWidgetLoadingPlaceholder from './ClusterDetailWidgetLoadingPlaceholder';
+
+const StyledRefreshableLabel = styled(RefreshableLabel)`
+  padding: 0;
+  margin: 0;
+`;
+
+interface IClusterDetailWidgetOptionalValueProps {
+  children: (value: string | number) => React.ReactElement;
+  value?: string | number;
+  replaceEmptyValue?: boolean;
+  loaderHeight?: number;
+  loaderWidth?: number;
+}
+
+const ClusterDetailWidgetOptionalValue: React.FC<IClusterDetailWidgetOptionalValueProps> = ({
```

Comment:
Yet another duplicate of similar, but slightly different optional value components. I'm still exploring different ways to write them, until I can come up with a well-written generic implementation